akka-docs-dev/rst/scala/http/routing-dsl/routes.rst

```diff
@@ -92,8 +111,6 @@ Consider this schematic example::
 
 Here five directives form a routing tree.
 
-.. rst-class:: wide
-
 - Route 1 will only be reached if directives ``a``, ``b`` and ``c`` all let the request pass through.
 - Route 2 will run if ``a`` and ``b`` pass, ``c`` rejects and ``d`` passes.
 - Route 3 will run if ``a`` and ``b`` pass, but ``c`` and ``d`` reject.
```

Comment:
the above example contains also a `Route 4`, also explain when it matches for completeness (when b doesn't).